LICENSE.txt

```diff
@@ -348,6 +348,29 @@ be/src/olap/lru_cache.cpp : BSD-style license
 
 --------------------------------------------------------------------------------
 
+be/src/olap/skiplist.h : BSD-style license
+
+  Copyright (c) 2011 The LevelDB Authors. All rights reserved.
+  Use of this source code is governed by a BSD-style license that can be
+  found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+--------------------------------------------------------------------------------
+
+be/src/util/arena.cpp : BSD-style license
+
+  Copyright (c) 2011 The LevelDB Authors. All rights reserved.
+  Use of this source code is governed by a BSD-style license that can be
+  found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+--------------------------------------------------------------------------------
+
+be/src/util/new_status.cpp : BSD-style license
```

Comment:
also new_status.h and arena.h

Reply:
OK, I will add this two files into LICENSE.txt